roachpb/data.go

```diff
@@ -275,23 +276,54 @@ func (t Timestamp) GoTime() time.Time {
 	return time.Unix(0, t.WallTime)
 }
 
+const (
+	checksumSize = 4
+	tagPos       = checksumSize
+	headerSize   = tagPos + 1
+)
+
+func (v *Value) checksum() uint32 {
+	if len(v.RawBytes) < checksumSize {
+		return 0
```

Comment:
Yes, a `nil` value (i.e. a deletion tombstone) will show up here. No other values with `len(v.RawBytes)` in the range `(0, checksumSize)` are possible because any actual value always has at least a checksum and a tag (5 bytes total).